csharp/Adapter/Microsoft.Spark.CSharp/Network/ByteBufPool.cs

```diff
@@ -21,6 +22,21 @@ internal sealed class ByteBufPool
         /// </summary>
         private const int MaxChunkSize = (int)((int.MaxValue + 1L) / 2);
 
+        /// <summary>
+        /// The default segment size to delimit a byte array chunk. 
+        /// </summary>
+        public const int DefaultSegmentSize = 65536; // 64K
+
+        /// <summary>
+        /// The default chunk order used to calculate chunk size and ensure it is a multiple of a segment size.
+        /// </summary>
+        public const int DefaultChunkOrder = 12; // 65536 << 12 = 256 MBytes per chunk
+
+        /// <summary>
+        /// The trial count to try allocation.
+        /// </summary>
+        private const int TrialsCount = 1000;
```

Comment:
Need more context on this constant as code comments - purpose, reason behind the magic number etc.